unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/NftTypeColorSupportingSO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using UnityEngine;
+
+[CreateAssetMenu(fileName = "NftTypeColorSupporting", menuName = "Variables/NftTypeColorSupporting")]
+public class NftTypeColorSupportingSO : ScriptableObject
+{
+    [SerializeField] public SerializableKeyValuePair<string, bool>[] nftTypes;
+
+    public bool GetTypeColorSupporting(string nftType)
```

Comment:
Can we rename it to something like "IsColorSupportedByType"? The current naming is not clear

Reply:
Done (https://github.com/decentraland/unity-renderer/pull/5028/commits/9d1f7544dfc7c5a924ce4532ccf3c6e24fff85ef).